### DISCUSSION TITLE
is it possible to have multiple wallet under one user?

### DISCUSSION THREAD

**Discussion by prisdax** [Web3Auth/Web3Auth#438] [Help]
is it possible to have multiple wallet under one user?

**Comment by yashovardhan** [comment]
Hey @prisdax 

Thanks for your question and sorry for the delay in reply from our side.

Yes it is possible to configure multiple wallet under a single user, but this has to be implemented from your side. Web3Auth provides you with a private key which you can utilise further as a seed for HDKey. This HDKey can be used to generate more keys while translates to more accounts.

You can utilise [this package](https://www.npmjs.com/package/hdkey) wherein the seed will be the private key exported from web3auth for the particular user. 

You can also try this demo which can help you understand this concept further: https://iancoleman.io/bip39/

This documentation page will help you understand how you can export user's private key in the frontend contexts: https://web3auth.io/docs/sdk/web/providers/other
